packages/synapse-react-client/package.json

```diff
@@ -200,11 +198,6 @@
     "chromatic": "^11.0.8",
     "copyfiles": "^2.4.1",
     "core-js": "^3.36.0",
-    "esbuild": "^0.20.1",
```

Comment:
Decided to drop esbuild after running into [this issue](https://github.com/evanw/esbuild/issues/1944) with a few of our dependencies. It may have been possible to work around it, but Vite was more simple to configure and will likely be less work to maintain.